docs/releases/1.9.txt

```diff
@@ -194,6 +196,15 @@ Miscellaneous
 Features deprecated in 1.9
 ==========================
 
+assignment_tag()
+~~~~~~~~~~~~~~~~
+
+Django 1.4 added the :ref:`assignment_tag<howto-custom-template-tags-assignment-tags>`
```

Comment:
prefer not linking to the deprecated thing as it's one less link to remove when thing is removed